app/javascript/Components/Result/text_viewer.jsx

```diff
@@ -37,6 +37,10 @@ export class TextViewer extends React.PureComponent {
   componentDidMount() {
     this.highlight_root = this.raw_content.current.parentNode;
 
+    if (localStorage.getItem("text_viewer_font_size") != null) {
+      this.setState({font_size: Number(localStorage.getItem("text_viewer_font_size"))});
```

Comment:
let's do the Number parsing in a try-catch; if the number fails to parse, don't set state and instead delete the localStorage value